trl/trainer/ppo_trainer.py

```diff
@@ -20,6 +20,7 @@
 from typing import List, Optional, Union
 
 import datasets
+import peft
```

Comment:
```suggestion
import datasets
from peft.peft_model import PeftModelForCausalLM
```

Reply:
Btw maybe the check can be better if we use `PeftModel` instead, so that it also works for seq2Seq models